packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -33,11 +35,16 @@ export type TagRevisionKarmaChange = KarmaChangeBase & {
   tagId: string,
 }
 
+export interface RecentVoteInfo extends DbVote {
+  postedAt: Date,
```

Comment:
This doesn't seem right; if you're only returning the fields specified in `voteFields`, then this won't have most of the fields on `DbVote` (or `postedAt`)

Reply:
Yeah I was pretty unsure how to handle this correctly, and I was figuring on pairing with you to iron out exactly how to handle the typing here.